fvm/fvm.go

```diff
@@ -18,9 +18,14 @@ type Procedure interface {
 	Run(vm *VirtualMachine, ctx Context, sth *state.StateHolder, programs *programs.Programs) error
 }
 
-func NewInterpreterRuntime() runtime.Runtime {
-	return runtime.NewInterpreterRuntime(
+func NewInterpreterRuntime(options ...runtime.Option) runtime.Runtime {
+
+	defaultOptions := []runtime.Option{
 		runtime.WithContractUpdateValidationEnabled(true),
+	}
+
+	return runtime.NewInterpreterRuntime(
+		append(defaultOptions, options...)...,
```

Comment:
@janezpodhostnik this is expected right?

Reply:
This change pretty much just allows passing additional options, doesn't change existing behaviour